### PR TITLE
Add customer parameter for java agent config "agent.namespace"

### PR DESCRIPTION
**Pull request by 861763765**
Add the customer parameter for java agent config "agent.namespace", so we can use the traceId from nginx to backend(java) server.

### CODE REVIEW

**Comment by 861763765** [comment]
close

**Comment by 861763765** [comment]
Add the customer parameter for java agent config "agent.namespace", so we can use the traceId from nginx to backend(java) server.
update tracer.lua span.lua traceing_context.lua

File: README.md [hunk 1]
```diff
@@ -47,8 +47,9 @@ http {
                 -- Please set them as service logic name or DNS name
                 --
                 -- Currently, we can not have the upstream real network address
+                -- The parameter "customer_namespace" for java agent config "agent.namespace"
                 ------------------------------------------------------
-                require("tracer"):start("upstream service")
+                require("tracer"):start("upstream service", "customer_namespace")
```
Comment (wu-sheng) [inline README.md:52]: This key should only be a optional parameter.

Comment (861763765) [reply README.md:52]: Yes, this parameter will be used only after we use the configuration of “agent.namespace” in the java agent. if we don't use it , this parameter can be nil

**Comment by wu-sheng** [comment]
#30 has higher priority, we need it to adopt 8.0 new protocol.

**Comment by wu-sheng** [comment]
#30 merged, please resolve conflicts and make the CI works.

**Comment by wu-sheng** [comment]
Any update?

**Comment by wu-sheng** [comment]
No update in a long time. Have to close for now.

**Pull request closed without merging**